### DISCUSSION TITLE
Is this a dead project?

### DISCUSSION THREAD

**Discussion by shakefu** [community/community#160691] [Actions]
No releases in 3 months, no real commits other than bumping the runner version in the same window... hundreds of open issues, nearly a hundred open PRs? Heck even their CODEOWNERS file is throwing errors because someone was removed from the Org? So is this out of feature development? Is it even in security maintenance?

**Comment by dongho-jung** [comment]
It’s already been over three months since someone asked about this... but now, it seems like there’s no one left to provide an answer anymore. -> [Is this project still alive?](https://github.com/actions/actions-runner-controller/discussions/3583)

When the community was developing and operating it independently, I appreciated the immediate bug responses and the transparent communication between maintainers and users. However, after the ownership was transferred to GitHub, it's been somewhat disappointing that this aspect has become harder to see.

If they were going to abandon the project like this, I wonder why they even took over the existing project and turned it into a legacy in the first place.

This might come across as somewhat paranoid and rude, but since we're no longer receiving any feedback from the maintainers, I dare to share my thoughts: I can’t help but wonder if they’re intentionally pushing for GitHub-hosted solutions because self-hosted ones aren't very profitable.

Because GitHub isn’t some small team, I just can’t understand how they could let the project be neglected like this.

I really hope that someday this project can return to how it used to be.

**Reply by shakefu** [reply]
I mean this is super useful for GitHub Enterprise customers, so I have a hard time imagining they're neglecting it because they're not profitable.

But yeah, this is really disappointing since the Scale Sets had so much promise for k8s based runners which actually scaled with load instead of just idling a bunch of runners all the time.
